### DISCUSSION TITLE
0.22 BUGS

### DISCUSSION THREAD

**Discussion by g0ftd** [egzumer/uv-k5-firmware-custom#558] [General]
Not a huge problem, but just for info.

Scrambler doesn't work.
Compander doesn't work.
Mic boost doesn't work.

Rig used = UV5R+ version.
They don't break the radio, but having them in the menu options is not worthwhile.

Tnx.

**Comment by nikant** [comment]
just out of curiosity what exactly doesn't work in Scrambler? 
(haven't tested the rest)

**Comment by g0ftd** [comment]
Hi,

I activated one menu which says SCR on the screen and also the option that
specifies the
frequency in HZ to use.

I then listened to the audio on another receiver and it was still
transmitting ordinary clear FM.

I hope this helps.

73 de Andy

**Reply by nikant** [reply]
Works fine in mine.. just tested 
https://youtu.be/38oCRA_G-qE

**Comment by g0ftd** [comment]
Interesting.
I'll have to do a re-test at some point in the future, just in case I have
missed anything.
But it is a low priority, since I don't really need the function, and I was
merely testing
some features for personal knowledge.

Personally, I like the EGZUMER 0.22 as it is, but I just wish for the full
options to set L/M/H power
settings like IJV.

73 de Andy

**Reply by g0ftd** [reply]
I retested the EGZUMER scambler - it works.

It requires two menu settings, access when you switch on the rig holding ptt+side button 1.

Set SCR on/off
and then scroll down to set the scrambler audio frequency.

**Comment by XiliX69** [comment]
Testing on a Quanshen K6. Option #59 "RX Mode" when changed to MAIN ONLY, the radio still appears in dual receive mode on the main screen.

**Reply by prokrypt** [reply]
that may be normal. I don't think there is single VFO display on this firmware
